Software/Frontend/app/src/main/java/hr/foi/air/baufind/ui/screens/JobRoom/JobRoom.kt

```diff
@@ -0,0 +1,54 @@
+package hr.foi.air.baufind.ui.screens.JobRoom
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Button
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import androidx.navigation.NavController
+import hr.foi.air.baufind.ui.components.PersonInRoomCard
+import hr.foi.air.baufind.ui.components.RoleInJobCard
+import hr.foi.air.baufind.ws.network.TokenProvider
+
+@Composable
+fun JobRoomScreen(navController: NavController,tokenProvider: TokenProvider,jobID: Int){
+    val viewModel: JobRoomViewModel = viewModel()
+    viewModel.tokenProvider.value = tokenProvider
+
+    LaunchedEffect(Unit) {
+        viewModel.getAllSkills()
+        viewModel.getJobRoom(jobID)
+        viewModel.loadJobPeople(jobID)
+    }
+    Column(
+        modifier = Modifier.fillMaxSize().padding(16.dp),
+    ) {
+        //promjenit
```

Comment:
Što promijeniti?

Reply:
Trebao bi dodati spinner i to za bolji UX ali to u sljedecem BAUFU